### PR TITLE
update readme example for running passed in  devnet

### PR DESCRIPTION
**Pull request by tomdong2010**
networkid =0xff   in devnet   , it can be passed, this modification  can be passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ var MyContract = web3.qkc.contract(abiArray);
 var contractInstance = MyContract.at(address);
 
 // deploy new contract
-var contractInstance = MyContract.new([constructorParam1] [, constructorParam2], {data: '0x12345...', from: myAccount, gas: 1000000});
+var contractInstance = MyContract.new([constructorParam1] [, constructorParam2], {data: '0x12345...', from: myAccount, gas: 1000000,gasPrice:1000000000,fromFullShardKey:fromFullShardKey,toFullShardKey:toFullShardKey});
```
Comment (ninjaahhh) [inline README.md]: add spaces between fields

File: index.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ if (typeof window === 'undefined') {
 } else {
   Web3 = window.Web3; // eslint-disable-line
 }
-
+//
```
Comment (ninjaahhh) [inline index.js]: remove

File: index.js [hunk 3]
```diff
@@ -245,8 +245,9 @@ export default {
           }
           if (obj.fromFullShardKey === undefined || obj.toFullShardKey === undefined) {
             throw new Error('`fromFullShardKey` and `toFullShardKey` are required');
-          }
 
+          }
+          //
```
Comment (ninjaahhh) [inline index.js]: remove

File: index.js [hunk 4]
```diff
@@ -274,6 +275,7 @@ export default {
           //    '0x1' typed encoding matching MetaMask initial implementation of EIP-712
           rawTx.version = '0x1';
 
+
```
Comment (ninjaahhh) [inline index.js]: remove extra space

File: index.js [hunk 5]
```diff
@@ -325,5 +327,8 @@ export default {
     web3in.eth.sendTransaction = web3in.qkc.sendTransaction.bind(web3in.qkc); // eslint-disable-line
     web3in.eth.getTransactionReceipt = web3in.qkc.getTransactionReceipt; // eslint-disable-line
     web3in.eth.getCode = web3in.qkc.getCode; // eslint-disable-line
+
```
Comment (ninjaahhh) [inline index.js]: remove extra space

File: index.js [hunk 6]
```diff
@@ -325,5 +327,8 @@ export default {
...
   },
+
```
Comment (ninjaahhh) [inline index.js]: same

File: index.js [hunk 7]
```diff
@@ -325,5 +327,8 @@ export default {
...
+
```
Comment (ninjaahhh) [inline index.js]: same

**Review by ninjaahhh — CHANGES_REQUESTED** [review]

File: index.js (hunk 7 above)
Comment (tomdong2010) [reply index.js]: ok

File: README.md (hunk 1 above)
Comment (tomdong2010) [reply README.md]: ok

File: README.md [hunk 8]
```diff
@@ -352,7 +352,7 @@ var MyContract = web3.qkc.contract(abiArray);
 var contractInstance = MyContract.at(address);
 
 // deploy new contract
-var contractInstance = MyContract.new([constructorParam1] [, constructorParam2], {data: '0x12345...', from: myAccount, gas: 1000000});
+var contractInstance = MyContract.new([constructorParam1] [, constructorParam2], {data: '0x12345...', from: myAccount, gas: 1000000,gasPrice: 1000000000,fromFullShardKey: fromFullShardKey, toFullShardKey: toFullShardKey});
```
Comment (ninjaahhh) [inline README.md]: add a space between `gas` and `gasPrice`, also the other fields

**Review by ninjaahhh — APPROVED** [review]

**Pull request merged**